### PR TITLE
Remove 'copy of' string in _create_lesson_copy()

### PR DESCRIPTION
**Pull request by ankitjavalkar**
- Remove Copy Of in Module, Lesson and Quiz names when duplicating courses

- Prevent student users from accessing Questions

### CODE REVIEW

**Comment by CruiseDevice** [comment]
@ankitjavalkar can you create the issue for this?

File: yaksh/models.py [hunk 1]
```diff
@@ -296,7 +296,7 @@ def _create_lesson_copy(self, user):
         lesson_files = self.get_files()
         new_lesson = self
         new_lesson.id = None
-        new_lesson.name = "Copy of {0}".format(self.name)
+        new_lesson.name = self.name
```
Comment (prathamesh920) [inline yaksh/models.py:299]: If the name will be same as original lesson name then no need of assigning the same name.

File: yaksh/models.py [hunk 2]
```diff
@@ -575,7 +575,7 @@ def _create_quiz_copy(self, user):
         question_papers = self.questionpaper_set.all()
         new_quiz = self
         new_quiz.id = None
-        new_quiz.description = "Copy of {0}".format(self.description)
+        new_quiz.description = self.description
```
Comment (prathamesh920) [inline yaksh/models.py:578]: Same here, not required

File: yaksh/models.py [hunk 3]
```diff
@@ -932,7 +932,7 @@ def create_duplicate_course(self, user):
         copy_course_name = "Copy Of {0}".format(self.name)
         new_course = self._create_duplicate_instance(user, copy_course_name)
         for module in learning_modules:
-            copy_module_name = "Copy of {0}".format(module.name)
+            copy_module_name = module.name
```
Comment (prathamesh920) [inline yaksh/models.py:935]: Same here

**Comment by ankitjavalkar** [comment]
Changes based on suggestions have been made

**Pull request merged**